ISSNow/iss_now.go

```
@@ -0,0 +1,89 @@
+package ISSNow
+
+import (
+	"encoding/json"
+	"github.com/kelvins/geocoder"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type IssRaw struct {
+	Timestamp   int `json:"timestamp"`
+	IssPosition struct {
+		Latitude  string `json:"latitude"`
+		Longitude string `json:"longitude"`
+	} `json:"iss_position"`
+	Message string `json:"message"`
+}
+
+type Iss struct {
+	Latitude  string
+	Longitude string
+	Location  string
+}
+
+var (
+	url    = "http://api.open-notify.org/iss-now.json"
+	client = http.Client{
+		Timeout: time.Second * 10,
+	}
+)
+
+func GetLocation(apikey string) Iss {
+	coords := getISSCoords()
+
+	latitude, _ := strconv.ParseFloat(coords.IssPosition.Latitude, 32)
+	longitude, _ := strconv.ParseFloat(coords.IssPosition.Longitude, 32)
```

You don't mind ignoring the errors? 😄